app/Console/ConcludeCommand.php

```diff
@@ -15,7 +15,7 @@
 class ConcludeCommand extends AbstractConcludeCommand
 {
     const TIMEZONE = 'Europe/Brussels';
-    const SOLR_DATE_TIME_FORMAT = 'Y-m-d\TH:i:s\Z';
+    const LUCENE_DATE_TIME_FORMAT = 'Y-m-d\TH:i:s\Z';
```

Comment:
I think this date format is not 100% correct for SAPI3.

The dates in SAPI3 are formatted as `yyyy-MM-dd'T'HH:mm:ssZZ`: https://github.com/cultuurnet/udb3-search-service/blob/master/src/ElasticSearch/Operations/json/mapping_event.json#L22

Elasticsearch uses this pattern syntax: https://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html

```
Symbol  Meaning                      Presentation  Examples
------  -------                      ------------  -------
Z       time zone offset/id          zone          -0800; -08:00; America/Los_Angeles

The count of pattern letters determine the format. 

Zone: 'Z' outputs offset without a colon, 'ZZ' outputs the offset with a colon, 'ZZZ' or more outputs the zone id. 
```

So the dates are formatted as for example: `2021-01-01T00:00:00+01:00`

In PHP, that is `Y-m-d\TH:i:sP`, not `Y-m-d\TH:i:s\Z`: https://www.php.net/manual/en/datetime.format.php

```
P | Difference to Greenwich time (GMT) with colon between hours and minutes | Example: +02:00
Z | Timezone offset in seconds. The offset for timezones west of UTC is always negative, and for those east of UTC is always positive. | -43200 through 50400
```

`Y-m-d\TH:i:sP` is the same as `DATE_ATOM`, which is also what we use as format when indexing the documents. For example: https://github.com/cultuurnet/udb3-search-service/blob/master/src/ElasticSearch/JsonDocument/Properties/AvailabilityTransformer.php#L59

Reply:
The `\` before `\Z` is also weird, because that escapes it and would keep the `Z` instead of putting the timezone there. (Like `\T`)

Reply:
Then I think it's best to just use `$lowerDateBoundary->format(DateTime::ATOM)`